src/PackageAPI.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace LibYear;
+
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\GuzzleException;
+
+class PackageAPI
+{
+	private ClientInterface $httpClient;
+
+	public function __construct(ClientInterface $httpClient, $stderr)
+	{
+		$this->httpClient = $httpClient;
+		$this->stderr = $stderr;
+	}
+
+	public function getPackageInfo(string $name, string $packageUrl): array
```

Comment:
Rather than passing in the whole URL, can we just pass in the repository base? Seems like within this class is the place to determine the correct URL